tests/sdk/test_util.py

```diff
@@ -10,3 +11,17 @@ def test_convert_timestamp_to_str_returns_expected_str():
 def test_convert_datetime_to_timestamp_str_returns_expected_str():
     d = datetime(2020, 4, 19, 13, 3, 2, 3)
     assert util.convert_datetime_to_timestamp_str(d) == "2020-04-19T13:03:02.000Z"
+
+
+class FilterClassTest(object):
```

Comment:
See this comment: https://github.com/code42/py42/pull/190/files#r485867556

This test is what prompted that. I don't think this class needs to be a `Filter` class. This util method does not care about that.  Any class would work... So, a less confusing test class here would be `TestClass(object)` and the test below would be `test_get_attribute_keys_from_class_returns_public_class_attribute_keys()`